packages/ui/core-components/src/lib/unsorted/viz/table/DataTable.stories.svelte

```diff
@@ -50,14 +55,46 @@
 		`SELECT * from flights where regulator in ('Afghanistan', 'Belgium', 'Canada', 'Denmark') limit 50`,
 		query
 	)}
-	<DataTable {data} title="Flights" search groupBy="regulator">
+	<DataTable
+		{data}
+		title="Flights"
+		search
+		groupBy="regulator"
+		sortBy="regulator"
+		sortAsc="true"
```

Comment:
this is inconsistent with the API on some of our other components, where you would do

`sort="column_name desc"`

Reply:
good point. I was thinking about that one, but thought it would be harder to give clear error messages. But I think consistency is more important, especially once we move the components to accepting more SQL fragments